app/controllers/task_managements_controller.rb

```diff
@@ -23,6 +54,20 @@ def update
     redirect_to dashboard_path
   end
 
+  def send_email_notifications(task)
+    notif_taskee = User.find_by_id(task.taskee_id)
+    notif_tasker = User.find_by_id(task.tasker_id)
```

Comment:
You can just use the `find` method directly instead of `find_by_id`. Reason been that `find_by_id` has been depreciated and `find` method finds a record by id(primary key). If you want to find by a different column, you can use `find_by` method eg `User.find_by(name: "Herbert")`